cirq/ops/common_gates.py

```diff
@@ -191,6 +191,9 @@ def _phase_by_(self, phase_turns, qubit_index):
             exponent=self._exponent,
             phase_exponent=phase_turns * 2)
 
+    def _has_stabilizer_effect_(self) -> bool:
+        return self.exponent % 1 == 0
```

Comment:
The exponent may be a symbol. If it's a symbol, return None. `if self._is_parameterized_(): ...`

Reply:
Done